src/msw/spinctrl.cpp

```diff
@@ -878,4 +878,20 @@ void wxSpinCtrl::DoClientToScreen(int *x, int *y) const
     wxWindow::MSWDoClientToScreen(GetBuddyHwnd(), x, y);
 }
 
+void wxSpinCtrl::SetIncrement(int value)
+{
+    UDACCEL accel;
+    accel.nSec = 0;
+    accel.nInc = value;
+    ::SendMessage( GetHwnd (), UDM_SETACCEL, 1, (LPARAM) &accel);
+}
+
+int  wxSpinCtrl::GetIncrement() const
+{
+    int increment;
+    UDACCEL accel;
+    ::SendMessage( GetHwnd(), UDM_GETACCEL, 1, (LPARAM) &accel );
+    increment = accel.nInc;
+    return increment;
```

Comment:
What's wrong with just
```suggestion
    return accel.nInc;
```
?